noir_stdlib/src/hash/sha256.nr

```diff
@@ -12,15 +12,25 @@ global MSG_SIZE_PTR = 56;
 // Size of the message block when packed as 4-byte integer array.
 global INT_BLOCK_SIZE = 16;
 
+// A `u32` integer consists of 4 bytes.
+global INT_SIZE = 4;
+
+// Index of the integer in the `INT_BLOCK` where the length is written.
+global INT_SIZE_PTR = MSG_SIZE_PTR / INT_SIZE;
+
 // Index of a byte in a 64 byte block; ie. 0..=63
 type BLOCK_BYTE_PTR = u32;
 
 // The foreign function to compress blocks works on 16 pieces of 4-byte integers, instead of 64 bytes.
 type INT_BLOCK = [u32; INT_BLOCK_SIZE];
 
+// We can represent 4-byte integers during construction as `Field` or `u32`.
+// Using `u32` because we shift right by dividing by 256, which doesn't work with `Field`.
+type INT_FIELD = u32;
```

Comment:
Let's remove this type alias, I don't think we're gaining much from it and it just makes it harder to see what type we're using.